src/apps/ipv4/arp.lua

```diff
@@ -143,11 +146,19 @@ function ARP:new(conf)
    return setmetatable(o, {__index=ARP})
 end
 
+function ARP:arp_resolving (ip)
+   print(("ARP: Resolving '%s'"):format(ipv4:ntop(self.next_ip)))
+   if self.alarm_notification then
+      local key = {resource='external-interface', alarm_type_id='arp-resolution'}
+      alarm_codec.raise_alarm(key)
```

Comment:
We should use that lovely Lua single-table-argument syntax:
```
alarm_codec.raise_alarm {
  resource='external-interface', alarm_type_id='arp-resolution'
}
```

Just a little comment for future patches :)